plugins/actions.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+package plugins
```

Comment:
What do you think about putting this as a subfolder of `experimental/` and not guaranteeing API compatibility for that package until some more testing?

Reply:
But I think we can guarantee API compatibility, we haven't changed any of these interfaces in more than a year

Reply:
We changed the factory registration method relatively recent for operator - looked at applying the same pattern for actions for consistency but there is some more tricky ordering involved there. It should be possible just didn't prioritize it yet.

The other bigger question is whether `string` is appropriate for operator vs `[]byte` - we reduced allocations a lot by wrapping / extracting unsafely but still there is allocation between steps no matter what when using `string`. I guess changing to `[]byte` can be a good idea but a big change.

Reply:
> The other bigger question is whether string is appropriate for operator vs []byte 

Will this be discussed/implemented in another PR?